Editor/Scripts/MeshSyncServerInspectorUtils.cs

```diff
@@ -196,6 +196,7 @@ internal static class MeshSyncServerInspectorUtils {
             if (server.m_DCCInterop != null) {
                 server.m_DCCInterop.runMode = previousRunMode;
                 server.m_DCCInterop.OpenDCCTool(asset);
+                server.IgnoreNextNewSession();
```

Comment:
Wouldn't it better to rename this to something along the line of 
`ForceDeleteChildrenInNextSession`
? 

If you agree, please rename the related variables as well

Reply:
Yeah, done.